packages/rspeedy/plugin-react/rslib.config.ts

```diff
@@ -10,6 +10,8 @@ export default defineConfig({
       'loaders/ignore-css-loader': './src/loaders/ignore-css-loader.ts',
       'loaders/invalid-import-error-loader':
         './src/loaders/invalid-import-error-loader.ts',
+      'background-only/error': './src/background-only/error.ts',
```

Comment:
Can we use the real [`background-only`](https://www.npmjs.com/background-only) npm package now?